cmd/node/init.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+	"github.com/machinefi/w3bstream-mainnet/enums"
+	"github.com/spf13/viper"
+	"log/slog"
+	"os"
+)
+
+func init() {
+	initStdLogger()
+	initEnvConfigBind()
+	initDatabaseMigrating()
+}
+
+func initStdLogger() {
+	var programLevel = slog.LevelDebug
+	h := slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{Level: programLevel})
+	slog.SetDefault(slog.New(h))
+}
+
+func initEnvConfigBind() {
+	viper.MustBindEnv(enums.EnvKeyServiceEndpoint)
+	viper.MustBindEnv(enums.EnvKeyRisc0ServerEndpoint)
+	viper.MustBindEnv(enums.EnvKeyHalo2ServerEndpoint)
+	viper.MustBindEnv(enums.EnvKeyProjectConfigPath)
+	viper.MustBindEnv(enums.EnvKeyChainEndpoint)
+	viper.MustBindEnv(enums.EnvKeyOperatorPrivateKey)
+	viper.MustBindEnv(enums.EnvKeyDatabaseDSN)
+}
+
+func initDatabaseMigrating() {
```

Comment:
Not the right place to init database. It is a component which may be implemented in a different way, so DB init is not part of the command.
`TODO: move out of cmd/node`